### PR TITLE
#499 Update to latest versions 

### LINKED ISSUE #499: Update to latest GLSP version due to API breaks
Update to latest GLSP, emfcloud and modelserver-glsp-integration version

### PR DESCRIPTION
**Pull request by ndoschek**
- Update backend to latest versions 
   - Update Model Server, GLSP and integration to latest versions
      - Adapt to API breaks from GLSP server (Refactoring of `OperationHandler` type hierarchy API)
   - Eclipse launch configurations
      - Use execution environment `JavaSE-17` instead of specific JRE container if installation differs
- Update client to latest versions 
   - Update to latest Theia, emfcloud and GLSP dependencies
      - Adapt to API breaks from GLSP
   - Switch to `peerDependencies` for Theia dependencies and remove resolutions block
   - Update `comparison-extension` to latest version
      - Consume icon and styling update

Resolves #499

### CODE REVIEW

File: client/coffee-workflow-analyzer/src/common/workflow-analyze-protocol.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019-2020 EclipseSource and others.
+ * Copyright (c) 2019-2022 EclipseSource and others.
```
Comment (eneufeld) [inline client/coffee-workflow-analyzer/src/common/workflow-analyze-protocol.ts:2]: why 2022?

File: client/package.json [hunk 2]
```diff
@@ -59,49 +59,6 @@
...
-    "**/@eclipse-glsp/client": "1.1.0-next.7ff1b00.184",
-    "**/@eclipse-glsp/protocol": "1.1.0-next.7ff1b00.184",
-    "**/@eclipse-glsp/theia-integration": "1.0.0-theia1.27.0",
-    "**/@eclipse-emfcloud/modelserver-client": "0.8.0-theia-cr01",
-    "**/@eclipse-emfcloud/modelserver-theia": "0.8.0-theia-cr01",
-    "**/@eclipse-emfcloud/theia-tree-editor": "0.7.0-next.4b9e870",
-    "**/sprotty": "0.12.0",
-    "**/sprotty-protocol": "0.12.0",
-    "**/sprotty-theia": "0.12.0"
+    "**/vscode-jsonrpc": "^8.1.0"
```
Comment (eneufeld) [inline client/package.json:62]: why don't we need any resolution block anymore?

File: client/coffee-workflow-analyzer/src/common/workflow-analyze-protocol.ts (hunk 1 above)
Comment (ndoschek) [reply client/coffee-workflow-analyzer/src/common/workflow-analyze-protocol.ts]: Thanks for catching that, I reverted that change as it was a leftover from a previous change which is not relevant anymore.

File: client/package.json (hunk 2 above)
Comment (ndoschek) [reply client/package.json]: Due to the `peerDependencies` we could get rid of the `@theia` resolutions and we can remove the remaining ones (mainly GLSP and emfcloud) as well as we can define the actual versions in the respective `package.json`. Mostly, they are only used by one package anyway, except for the `modelserver-theia`. If you would like to keep the resolution for that package, it is also fine by me.

Comment (eneufeld) [reply client/package.json]: I'm happy to get rid of it.

**Review by eneufeld — APPROVED** [review]
Thank you!

**Pull request merged**